examples/user_guide/12-Responding_to_Events.ipynb

```diff
@@ -344,26 +344,79 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from holoviews.streams import Params\n",
+    "class BankAccount(param.Parameterized):\n",
+    "    balance = param.Number(default=0, doc=\"Bank balance in USD\")\n",
+    "    overdraft = param.Number(default=200, doc=\"Overdraft limit\")\n",
     "\n",
-    "class Style(param.Parameterized):\n",
-    "\n",
-    "    cmap = param.ObjectSelector(default='viridis', objects=['viridis', 'plasma', 'magma'])\n",
+    "account = BankAccount(name='Jane', balance=300)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can link parameter changes straight to DynamicMap callable parameters using the streams dictionary format (HoloViews version >= 1.14.2):"
```

Comment:
```suggestion
    "You can link parameter changes straight to DynamicMap callable parameters by passing a stream_param:param dictionary to the `streams` argument (for HoloViews version >= 1.14.2):"
```

Reply:
I'm not sure `stream_param:param` dictionary is clear. It is more `keyword:param` imho for the callable keywords. I'll apply this suggestion though and edit after.